ramls/oai-pmh.raml

```diff
@@ -0,0 +1,131 @@
+          body:
+            application/xml:
+              schema: oai-pmh
+              example: !include examples/no_records_error.sample
+
+    /{id}:
+      uriParameters:
+        id:
+          description: The unique identifier of the item in the repository
+          type: string
+      get:
+        description: Retrieve record with given Id
+        is: [ metadataPrefix ]
+        responses:
+          200:
+            description: "Returns item with a given ID"
+            body:
+              application/xml:
+                example: !include examples/record.sample
+                schema: oai-pmh
+          404:
+            description: "Item with a given ID and selective harvesting params (if any) not found"
+            body:
+              application/xml:
+                schema: oai-pmh
+                example: !include examples/no_record_error.sample
+          500:
+            description: "Internal server error, e.g. due to misconfiguration"
+            body:
+              text/plain:
+                example: "Internal server error, contact administrator"
+
+  /identifiers:
+    displayName: Identifiers
+    type:
+      list:
+        exampleCollection: !include examples/identifiers.sample
+    get:
+      is: [ partionable, selective, metadataPrefix ]
+      responses:
+        404:
+          description: "Item with a given ID and selective harvesting params (if any) not found"
+          body:
+            application/xml:
+              schema: oai-pmh
+              example: !include examples/no_records_error.sample
+
+  /metadata_formats:
+    displayName: MetadataFormats
+    type:
+      list:
+        exampleCollection: !include examples/metadata_formats.sample
+    get:
+      queryParameters:
+        identifier:
+          description: 'unique identifier of the item in the repository from which the record must be disseminated'
+          type: string
+          required: false
+      responses:
+        404:
+          description: "Item with a given ID and selective harvesting params (if any) not found"
+          body:
+            application/xml:
+              schema: oai-pmh
+              example: !include examples/no_record_error.sample
+        422:
+          description: "The request was well-formed, but contains semantic errors"
+          body:
+            application/xml:
+              schema: oai-pmh
+              example: !include examples/no_metadata_format_error.sample
+
+  /sets:
+    displayName: Sets
+    type:
+      list:
+        exampleCollection: !include examples/sets.sample
+    get:
+      is: [ partionable ]
+
+  /repository_info:
+    displayName: RepositoryInfo
+    get:
+      description: Returns information about a repository
+      responses:
+        200:
+          description: "Returns repository info"
+          body:
+            application/xml:
+              schema: oai-pmh
+              example: !include examples/repository_info.sample
```

Comment:
It makes sense to follow note from [RMB ugrading notes](https://github.com/folio-org/raml-module-builder/blob/master/doc/upgrading.md):
>10. The resource type examples must not be strict (will result in invalid json content otherwise)
        CHANGE:
```yml
            example: <<exampleItem>>
```
        TO:
```yml
            example:
                strict: false
                value: <<exampleItem>>
```

So all the samples in the raml will be like 
```yml
              example:
                strict: false
                value: !include examples/repository_info.sample
```

Reply:
I agree.
Already done.